Example-Audit-REXX/RXAUDIT1.rexx

```diff
@@ -0,0 +1,2478 @@
+/*  DESCRIPTIVE NAME:                                                  */
+/*    Sample REXX code that uses HWIREST API to audit LPAR             */
+/*    information.  It also takes advantage of the JSON Parser         */
+/*    to retrieve various pieces of the information.                   */
+/*                                                                     */
+/* OPERATION:                                                          */
+/*                                                                     */
+/*  CODE FLOW in this sample:                                          */
+/*    Call List CPCs to retrieve uri and target name for the specific  */
+/*         CPC                                                         */
+/*    Call List Logical Partitions of CPC to retrieve uri and          */
+/*         target name for the specific LPAR                           */
+/*    Call Get LPAR Properites for 'operating' LPARs to retrieve a     */
+/*         set of specific LPAR attributes                             */
+/*    Write collected content out to the provided partitioned data set */
+/*                                                                     */
+/* INVOCATION:                                                         */
+/*     RXAUDIT1 -D outputDataSet [-C CPCname] [-I] [-V]                */
+/*                                                                     */
+/*     Required input parameters:                                      */
+/*        -D <data set name> is the name of a pre-existing partitioned */
+/*           data set, if the LPAR audit is successful, a member       */
+/*           containing the audit information in CSV format will be    */
+/*           stored into the data set, the member will either be named */
+/*           LOCAL or the CPC name provided via -C option              */
+/*     Optional input parameters:                                      */
+/*        -C <CPCname> name of the CPC to audit, default is LOCAL CPC  */
+/*        -I indicate running out of ISV REXX environment,             */
+/*           default is TSO/E                                          */
+/*        -V turn on additional verbose JSON tracing                   */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXAUDIT1))'                             */
+/*            '-D HWI.AUDIT -I'                                        */
+/*            data set is HWI.AUDIT, run out of ISV REXX,              */
+/*            if successful will create HWI.AUDIT(LOCAL)               */
+/*                                                                     */
+/* DEPENDENCIES                                                        */
+/*     none.                                                           */
+/*                                                                     */
+/*    NOTES:                                                           */
```

Comment:
You might want to move NOTES: to line up with DEPENDENCIES and similarly the "No recovery logic has been supplied in this sample." should line up with the "none." under DEPENDENCIES

Reply:
done